## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 442189`
Commit: [`9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b`](https://github.com/NixOS/nixpkgs/commit/9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/9ddd3cb2a966ef2b1a378ccbfe0c6f3aaeb8d46b..pull/442189/head))
Merge: [`0f0c9e2bf07b5b8e093e58de468e8b5758bbb661`](https://github.com/NixOS/nixpkgs/commit/0f0c9e2bf07b5b8e093e58de468e8b5758bbb661)

Logs: https://github.com/sikmir/nixpkgs-review-gha/actions/runs/18050170324


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 27 packages built:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>home-assistant-component-tests.iskra</li>
    <li>home-assistant-component-tests.modbus</li>
    <li>home-assistant-component-tests.smarty</li>
    <li>home-assistant-component-tests.stiebel_eltron</li>
    <li>home-assistant-custom-components.solax_modbus</li>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>


---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 27 packages built:</summary>
  <ul>
    <li>expliot</li>
    <li>expliot.dist</li>
    <li>home-assistant-component-tests.iskra</li>
    <li>home-assistant-component-tests.modbus</li>
    <li>home-assistant-component-tests.smarty</li>
    <li>home-assistant-component-tests.stiebel_eltron</li>
    <li>home-assistant-custom-components.solax_modbus</li>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python312Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 6 warnings, 16 errors in 69.68s (0:01:09) =======</pre>
</details>
<details>
<summary>python313Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 5 warnings, 16 errors in 69.57s (0:01:09) =======</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 20 packages failed to build:</summary>
  <ul>
    <li>python312Packages.pyiskra</li>
    <li>python312Packages.pyiskra.dist</li>
    <li>python312Packages.pymodbus</li>
    <li>python312Packages.pymodbus.dist</li>
    <li>python312Packages.pysmarty2</li>
    <li>python312Packages.pysmarty2.dist</li>
    <li>python312Packages.pystiebeleltron</li>
    <li>python312Packages.pystiebeleltron.dist</li>
    <li>python312Packages.python-dali</li>
    <li>python312Packages.python-dali.dist</li>
    <li>python313Packages.pyiskra</li>
    <li>python313Packages.pyiskra.dist</li>
    <li>python313Packages.pymodbus</li>
    <li>python313Packages.pymodbus.dist</li>
    <li>python313Packages.pysmarty2</li>
    <li>python313Packages.pysmarty2.dist</li>
    <li>python313Packages.pystiebeleltron</li>
    <li>python313Packages.pystiebeleltron.dist</li>
    <li>python313Packages.python-dali</li>
    <li>python313Packages.python-dali.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python312Packages.pymodbus</summary>
<pre>FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TCP-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.TLS-localhost] - assert False
FAILED transport/test_reconnect.py::TestTransportReconnect::test_listen_disconnect[CommType.UDP-localhost] - assert False
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_receive_data - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_server_trace_connect_disconnect - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_connection_lost - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_shutdown_connection - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_no_device - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tcp_server_modbus_error - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_tls_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_tls_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_start_udp_server - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_close - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_serve_forever_twice - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_roundtrip - RuntimeError: Could not start listen, please check address.
ERROR server/test_server_asyncio.py::TestAsyncioServer::test_async_udp_server_exception - RuntimeError: Could not start listen, please check address.
====== 23 failed, 1426 passed, 5 warnings, 16 errors in 62.70s (0:01:02) =======</pre>
</details>
</details>

--- a/pkgs/development/python-modules/pyiskra/default.nix
+++ b/pkgs/development/python-modules/pyiskra/default.nix
@@ -9,14 +9,14 @@
 
 buildPythonPackage rec {
   pname = "pyiskra";
-  version = "0.1.26";
+  version = "0.1.27";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "Iskramis";
     repo = "pyiskra";
     tag = "v${version}";
-    hash = "sha256-RUMMXlLdG4cqBrNOK5f1jv0jSU8n/P3XrvA7l1hij1g=";
+    hash = "sha256-qPyspRYXolndNMLHG2Ln6eL0emRCYYr/00WhrX876k0=";
   };
 
   build-system = [ setuptools ];

--- a/pkgs/development/python-modules/pymodbus/default.nix
+++ b/pkgs/development/python-modules/pymodbus/default.nix
@@ -21,15 +21,17 @@
 
 buildPythonPackage rec {
   pname = "pymodbus";
-  version = "3.11.1";
+  version = "3.11.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "pymodbus-dev";
     repo = "pymodbus";
     tag = "v${version}";
-    hash = "sha256-wanWE0FDaTtIR2V5pjd7Vffp2C8qGYMyMrdG6pHFspo=";
+    hash = "sha256-xfulpF6NrHM+qJZSlTW4Wb0GqYXQytepyB6bjy2eI9Y=";
   };
+
+  __darwinAllowLocalNetworking = true;
 
   build-system = [ setuptools ];
 

